Tests/RoslynPluginGeneratorTests/AnalyzerPluginGeneratorTests.cs

```diff
@@ -441,18 +680,33 @@ public void CreatePluginManifest_Owners_NotAvailable()
 
         #region Private methods
 
-        private static ProcessedArgs CreateArgs(string packageId, string packageVersion, string language, string sqaleFilePath, bool acceptLicenses, string outputDirectory)
+        private static ProcessedArgs CreateArgs(string packageId, string packageVersion, string language, string sqaleFilePath, bool acceptLicenses, bool recurseDependencies, string outputDirectory)
         {
             ProcessedArgs args = new ProcessedArgs(
                 packageId,
                 new SemanticVersion(packageVersion),
                 language,
                 sqaleFilePath,
                 acceptLicenses,
+                recurseDependencies,
                 outputDirectory);
             return args;
         }
 
+        private void clearDirectory(string path)
```

Comment:
Incorrect casing in the method name.

Use [Directory.Delete(path, recurse)](https://msdn.microsoft.com/en-us/library/fxeahc5f(v=vs.110).aspx)